pkgs/by-name/op/opengist/package.nix

```diff
@@ -7,29 +7,22 @@
   moreutils,
   jq,
   git,
+  stdenv,
```

Comment:
Please, put `stdenv` at the top, right after `lib`.

Reply:
done